models/submissionsModel.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../database/dbConfig");
+
+// not sure if I need async here or not either
+function find() {
+  return db("application_submissions");
+}
```

Comment:
We don't need to access the entire submissions table so we can delete this function.